e2e/cases/resolve/with-main-fields/.gitignore

```diff
@@ -0,0 +1,2 @@
+!node_modules
```

Comment:
useless?

Reply:
This file is aim to keep node_modules folder exist to test resolve logic